src/helper/crud/crud.factory.ts

```diff
@@ -30,6 +30,10 @@ export function BaseCrudFactory<
   class Dto extends dto {}
   class UpdateDto extends PartialType(Dto) {}
   class QueryDto extends IntersectionType(PagerDto, PartialType(Dto)) {}
+  
+  Object.defineProperty(Dto, 'name', { value: pluralizeName })
+  Object.defineProperty(UpdateDto, 'name', { value: pluralizeName + 'UpdateDto' })
+  Object.defineProperty(QueryDto, 'name', { value: pluralizeName + 'QueryDto' })
```

Comment:
```suggestion
  class Dto extends dto {
    static name = upperFirst(`${pluralizeName}Dto`)
  }
  class UpdateDto extends PartialType(Dto) {
    static name = upperFirst(`${pluralizeName}UpdateDto`)
  }
  class QueryDto extends IntersectionType(PagerDto, PartialType(Dto)) {
    static name = upperFirst(`${pluralizeName}QueryDto`)
  }
```
感谢修正！这里使用 `static`关键字或许更简洁一些，并且代码相关性相对集中。

Reply:
受教了，学到新知识